.github/workflows/node.js.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.0.0, 18.6.0, 18.x, 20.0.0, 20.1.0, 20.x]
+        node-version: [18.0.0, 18.6.0, 18.x, 20.0.0, 20.1.0, 20.x, 22-nightly] # todo: use 22.x when released
```

```suggestion
        node-version: [18.0.0, 18.6.0, 18.x, 20.0.0, 20.1.0, 20.x, 22.0.0, 22.x]
```

use 22 when released